chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -160,6 +161,11 @@ def self.claimed(node,owner)
           end
         end
 
+        def self.multipath?(device = @device)
```

Comment:
Woops! Forgot to remove it, good catch!

Also great idea on the unit testing, will add a couple of tests to this new functions instead of only doing the unclaimed method.